Library/DataSource/UIView-Extensions.swift

```diff
@@ -1,30 +1,33 @@
 import UIKit
 
 private func swizzle(_ v: UIView.Type) {
-  [(#selector(v.traitCollectionDidChange(_:)), #selector(v.ksr_traitCollectionDidChange(_:)))]
-    .forEach { original, swizzled in
+  [
+    (#selector(v.traitCollectionDidChange(_:)), #selector(v.ksr_traitCollectionDidChange(_:))),
+    (#selector(v.layoutSubviews), #selector(v.ksr_layoutSubviews))
```

Comment:
This is the attempt at working around the change to how `traitCollectionDidChange(_:)` is called for `UIView`, some more detail here: https://medium.com/@anandin02/embracing-dark-mode-ios-13-29b29d55122b

I've tried using `layoutSubviews()` for now, it seems to improve things a lot but I'm not sure if it is the best solution, I still need to dig into it a bit more. Nevertheless, if it needs to this will probably change with the PR addressing the snapshot tests.